app/home/controllers.js

```diff
@@ -11,26 +13,37 @@ function dashboard(req, res) {
     return res.redirect('/moves')
   }
 
+  const week = dateHelpers.getCurrentWeekAsRange()
+  const [today] = dateHelpers.getCurrentDayAsRange()
+
   const sections = {
     outgoing: {
       permission: 'moves:view:outgoing',
       heading: 'dashboard::sections.outgoing.heading',
       filter: req.filterOutgoing,
+      time: formatDate(today),
```

Comment:
There is a [`helper`](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/27474d4c2c020708d358a52f6a9d1f7513f658d9/config/nunjucks/filters.js#L194) that will already combine the day and day if it’s either today or tomorrow.

Reply:
I have to wrap the `(today)` in a `<span>` to change the colour to grey, so I can't use the helper. I tried really hard to reuse what was already there but opted for the code above in the end.

Reply:
Ah yes. Of course!

Reply:
I wonder how @Andy-Millar feels about not having the relative value in grey...?

Reply:
I worked around the issue and found a way to reuse the existing helpers as well as the `<time>` tag.